pkgs/by-name/qd/qdiskinfo/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  smartmontools,
+  fetchFromGitHub,
+  cmake,
+  qt6,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "QDiskInfo";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "edisionnano";
+    repo = "QDiskInfo";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-YLNpQOmh/exBJDuYRUEvzNv/El4x+PttEoj+aCur9wg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qt6.full
```

Comment:
is this needed? Isn't there a small set of subpackages?

Reply:
It would appear so. The derivation builds with just qtbase.